environments/extractDiffs.toml

```diff
@@ -227,6 +231,10 @@
         deployed = 8925077
+    [contract.MEDIAN_YFI_1_1_14]
```

Comment:
Also noticing that it appears this address wasn't in this config or the backfill configs - I wonder if we might want to include it in the backfill events transformers' config (and later add it to the backfill storage process)?